### PR TITLE
EVEREST-986-fix-warning-message-for-TLS-certificate

### PR DESCRIPTION
**Pull request by daninda**

### CODE REVIEW

File: ui/apps/everest/src/pages/settings/monitoring-endpoints/createEditModal/create-edit-modal.tsx [hunk 1]
```diff
@@ -1,18 +1,18 @@
-import { useMemo } from 'react';
 import { TextInput } from '@percona/ui-lib';
 import { FormDialog } from 'components/form-dialog';
-import TlsCheckbox from 'components/tls-checkbox';
 import TlsAlert from 'components/tls-alert';
+import TlsCheckbox from 'components/tls-checkbox';
+import { useMemo } from 'react';
+import { AutoCompleteSelectAll } from '../../../../components/auto-complete-select-all/auto-complete-select-all';
```
Comment (solovevayaroslavna) [inline ui/apps/everest/src/pages/settings/monitoring-endpoints/createEditModal/create-edit-modal.tsx]:
```suggestion
import { AutoCompleteSelectAll } from 'components/auto-complete-select-all/auto-complete-select-all';
```

File: ui/apps/everest/src/pages/settings/monitoring-endpoints/createEditModal/create-edit-modal.tsx [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
+import { useNamespaces } from '../../../../hooks/api/namespaces/useNamespaces';
```
Comment (solovevayaroslavna) [inline ui/apps/everest/src/pages/settings/monitoring-endpoints/createEditModal/create-edit-modal.tsx]:
```suggestion
import { useNamespaces } from 'hooks/api/namespaces/useNamespaces';
```

File: ui/apps/everest/src/pages/settings/monitoring-endpoints/createEditModal/create-edit-modal.tsx [hunk 3]
```diff
@@ -1,18 +1,18 @@
-import { useMemo } from 'react';
```
Comment (solovevayaroslavna) [inline ui/apps/everest/src/pages/settings/monitoring-endpoints/createEditModal/create-edit-modal.tsx:1]: why we need to cut this part of code and add the same below? please try pnpm format in repo before commit

File: ui/apps/everest/src/pages/settings/monitoring-endpoints/createEditModal/create-edit-modal.tsx [hunk 4]
```diff
@@ -1,18 +1,18 @@
...
+import { useNamespaces } from '../../../../hooks/api/namespaces/useNamespaces';
+import { Messages } from '../monitoring-endpoints.messages';
 import {
   CreateEditEndpointModalProps,
   EndpointFormFields,
   EndpointFormType,
   endpointDefaultValues,
   getEndpointSchema,
 } from './create-edit-modal.types';
-import { Messages } from '../monitoring-endpoints.messages';
```
Comment (solovevayaroslavna) [inline ui/apps/everest/src/pages/settings/monitoring-endpoints/createEditModal/create-edit-modal.tsx:13]: the same question about messages, why we change the position of import? is this result of some command?

**Review by solovevayaroslavna — CHANGES_REQUESTED** [review]

File: ui/apps/everest/src/pages/settings/monitoring-endpoints/createEditModal/create-edit-modal.tsx (hunk 4 above)
Comment (daninda) [reply ui/apps/everest/src/pages/settings/monitoring-endpoints/createEditModal/create-edit-modal.tsx:13]: Its just the Prettier extension and formatting on save

File: ui/apps/everest/src/pages/settings/monitoring-endpoints/createEditModal/create-edit-modal.tsx (hunk 3 above)
Comment (daninda) [reply ui/apps/everest/src/pages/settings/monitoring-endpoints/createEditModal/create-edit-modal.tsx:1]: Sorry, i wont do this again(

**Pull request closed without merging**